src/Raven.Server/Documents/ETL/Providers/Raven/RavenEtlDocumentTransformer.cs

```
@@ -344,6 +432,139 @@ private void AddSingleCounterGroup(LazyStringValue docId, BlittableJsonReaderObj
+                    throw new InvalidOperationException($"Return type of `{function}` function should be a boolean or object. docId({docId}), timeSeriesName({timeSeriesName})");
+                }
+                else
+                {
+                    var toLoadLocal = (From: DateTime.MinValue, To: DateTime.MaxValue);
+                    foreach ((JsValue jsValue, PropertyDescriptor propertyDescriptor) in scriptRunnerResult.Instance.AsObject().GetOwnProperties())
+                    {
+                        var key = jsValue.AsString();
+                        switch (key)
+                        {
+                            case "from":
+                            case "From":
+                                if (toLoadLocal.From != DateTime.MinValue)
+                                    throw new InvalidOperationException($"Duplicate of property `From`/`from`. docId({docId}), timeSeriesName({timeSeriesName}), function({function})");
+                                toLoadLocal.From = propertyDescriptor.Value.AsDate().ToDateTime();
+                                break;
+                            case "to":
+                            case "To":
+                                if (toLoadLocal.To != DateTime.MaxValue)
+                                    throw new InvalidOperationException($"Duplicate of property `To`/`to`. docId({docId}), timeSeriesName({timeSeriesName}), function({function})");
+                                toLoadLocal.To = propertyDescriptor.Value.AsDate().ToDateTime();
+                                break;
+                            default:
+                                //TODO Maybe just ignore. Can be useful if we will add property in future and someone will run it with current version
+                                throw new InvalidOperationException($"Returned object should contain only `from` and `to` property but contain `{key}`. docId({docId}), timeSeriesName({timeSeriesName}), function({function})");
```

if we add something in the future then we'll have to add a handling for that here anyway. I don't see the point to silently ignore the value here